packages/components/inputs/async-creatable-select-input/README.md

```diff
@@ -83,6 +83,7 @@ export default Example;
 | `menuPortalTarget`                | `SafeHTMLElement`                                                                                  |          |         | Dom element to portal the select menu to                                                                                                                                                                                                                                                    |
 | `menuPortalZIndex`                | `number`                                                                                           |          | `1`     | z-index value for the menu portal                                                                                                                                                                                                                                                           |
 | `menuShouldBlockScroll`           | `bool`                                                                                             |          |         | whether the menu should block scroll while open                                                                                                                                                                                                                                             |
+| `closeMenuOnSelect`               | `bool`                                                                                             |          | `true`  | whether the menu should be closed on option select open                                                                                                                                                                                                                                     |
```

Comment:
Did you edit this file by hand?
There is this `generate-readmes` yarn command that handles the generation of this files automatically so its easier to handle and descriptions are the same in code comments and in these files.